modules/parrot_drush/manifests/components/php/drush_alias.pp

```diff
@@ -0,0 +1,25 @@
+define parrot_drush::components::php::drush_alias (
+  $root,
+  $ensure = 'present',
+  $uri = $name,
+  $php_executable = undef,
+  $protocol = '',
+)
+{
+
+  file {"/home/vagrant/.drush/$name.aliases.drushrc.php":
+    ensure => $ensure,
+    owner => 'root',
```

Comment:
Presumably the owner and group should be `vagrant` if these aliases are for the vagrant user.

Reply:
@darthsteven you mentioned the `www-data` user on https://github.com/computerminds/parrot/issues/140#issuecomment-389560544 , but when might that user need drush aliases? The only time I've ever wanted aliases for a user in parrot is just for the regular `vagrant` user, simply because that's the user I happen to get logged in as when running `parrot ssh`. Is it better to be running drush commands as a different user anyway, e.g. `www-data`?

So I'm also not sure we actually want aliases for the root user either.